src/desktop/apps/auction/components/layout/auction_info/AuctionInfoDesktop.tsx

```diff
@@ -34,16 +39,23 @@ function AuctionInfoDesktop(props) {
               <div className={b("upcomingLabel")}>{upcomingLabel}</div>
             )}
 
-            {showAddToCalendar && <AddToCalendarView event={event} />}
+            {showAddToCalendar && (
+              <AddToCalendar
+                startDate={liveStartAt || startAt}
+                endDate={endDate}
+                title={name}
```

Comment:
The name of the calendar event should reflect that it's the sale going live or closing vs. just being the sale name since I think that'll confuse users about when they can actually place a bid. E.g. something like "$SALE_NAME Goes Live" or "$SALE_NAME Closes" -- I'd work with the auctions team on the copy.

Reply:
The current change preserves the existing title implementation, is it ok if I add that to the list of improvements for later?

Reply:
Yep sounds good!